Sources/Views/AvatarView.swift

```diff
@@ -153,7 +153,7 @@ open class AvatarView: UIImageView {
     }
 
     required public init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+        super.init(coder: aDecoder)
```

Comment:
@talanov Please change implementation like below:
```
    required public init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
		prepareView()
    }
```

BTW, could you please fix other similar issues of other views?

Reply:
Done.

Reply:
@talanov, seems you still don't do what I want, please update referred my comment.